### PR TITLE
Widgetsnbextension: throw error on failure to render

### PR DESCRIPTION
**Pull request by martinRenou**
This goes in pair with https://github.com/jupyter/notebook/pull/6181

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
+1 on this.

This will enable viewing alternative mime types when widgets cannot be rendered. It would be interesting to have a independent release of widgetsnbextension with that change.

File: widgetsnbextension/src/extension.js [hunk 1]
```diff
@@ -123,15 +123,12 @@ function register_events(Jupyter, events, outputarea) {
       Jupyter.notebook.kernel &&
       Jupyter.notebook.kernel.widget_manager;
     if (!manager) {
-      node.textContent =
-        'Error rendering Jupyter widget: missing widget manager';
-      return;
+      throw 'Error rendering Jupyter widget: missing widget manager';
```
Comment (vidartf) [inline widgetsnbextension/src/extension.js:126]: Note: In general in Javascript you'd want to throw an Error, and not a string directly. Was this intentional?

Comment (martinRenou) [reply widgetsnbextension/src/extension.js:126]: Thanks for the review, I will update this

**Comment by martinRenou** [comment]
TODO

- What happens for current Notebook?
- Should we do a Notebook version check?

**Comment by martinRenou** [comment]
Now doing a Notebook version check to **not** change the behavior for Notebook 6.4.4 and before.

**Comment by SylvainCorlay** [comment]
Discussed during the ipywidgets call. Merging as decided in the call.

**Comment by SylvainCorlay** [comment]
@meeseeksdev backport to 7.x

**Pull request merged**